docs/database/readme.md

```diff
@@ -0,0 +1,159 @@
+# Schema
+
+```mermaid
+---
+title: Airneis database model
+---
+
+classDiagram
+
+Categories <|-- Products
+Carts <|-- Status
+Products <|-- Carts
+Users <|-- Addresses
+Users <|-- Carts
+Carts <|-- CartsProducts
+Products <|-- CartsProducts
+Products <|-- ProductImages
+Materials <|-- ProductsMaterials
+Products <|-- ProductsMaterials
+
+class Users {
+    id increments notNull
+    fullName text notNull
+    email text unique notNull
+    isAdmin boolean default false notNull
+    isActive boolean default false notNull
+}
+
+class Categories {
+    id increments notNull
+    name text notNull
+    thumbnailUrl text notNull
+    slug text unique notNull
+    orderOfDisplay number notNull
+}
+
+class Products {
+    id increments notNull
+    name text notNull
+    slug unique notNull
+    categoryId number notNull
+    price number notNull
+    description text notNull
+    thumbnailUrl text notNull
+    outOfStock boolean default false notNull
+    isHighlanderOfTheMoment boolean default false notNull
+}
+
+class ProductImages {
+    id increments notNull
+    productId number notNull
+    imageUrl text notNull
+}
+
+class Materials{
+    id increments notNull
+    name text notNull
+}
+
+class ProductsMaterials {
```

Comment:
What is the use of `ProductsMaterials`? When getting a product's material, can't we just add in the class Products : 
`materials Materials notNull` 

What do you think ?